src/main/java/jp/classmethod/aws/gradle/cloudformation/AmazonCloudFormationPluginExtension.java

```diff
@@ -173,6 +192,25 @@ public String findStackParameterValue(List<Parameter> cfnStackParameters, String
 		return param.get().getParameterValue();
 	}
 	
+	public String getStackOutputValue(String key) {
+		return findStackOutputValue(getStackOutputs(), key);
+	}
+	
+	public String getStackOutputValue(String stackName, String key) {
+		return findStackOutputValue(getStackOutputs(stackName), key);
+	}
+	
+	public String findStackOutputValue(List<Output> cfnStackOutputs, String key) {
+		Optional<Output> output = cfnStackOutputs.stream()
+			.filter(p -> p.getOutputKey().equals(key))
+			.findAny();
+		if (output.isPresent() == false) {
```

Comment:
Had some time yesterday to try this.  Just created #94 with my changes.  Think it's a step in the right direction but not sure if it's what @tellary and @dai0304 had in mind.